sootty/utils.py

```diff
@@ -0,0 +1,14 @@
+from math import ceil, log
+
+# Convert a decimal into any base (2-33)
```

Comment:
Why does it not work up to base 36? (I assume we can use a-z and 0-9 to get all 36 digits)

Reply:
Previously, if converting radix in `_wire_to_svg`, then after converting, in `type_from_value`, if "x" is in str(value), the type will be determined as Visualizer.ValueType.X rather than Visualizer.ValueType.DATA. However, since converting radix is done in `_value_to_svg`, it seems that the problem no longer exists because checking type by `type_from_value` is done before converting. I'll push a fix to make it work up to base 36.

Reply:
Now the radix could be 2 - 36. Tests have been passed.